sdk/core/azure-core/inc/azure/core/http/http.hpp

```diff
@@ -473,12 +490,19 @@ namespace Azure { namespace Core { namespace Http {
     }
 
     /**
-     * @brief Add an HTTP header.
+     * @brief Adds an HTTP header to the Request.
      *
-     * @param name HTTP header name.
-     * @param value HTTP header value.
+     * @param name The name for the header to be added.
+     * @param value The value for the header to be added.
+     * @param invalidHeaderIndex Optional size_t pointer to reference the first invalid character
+     * when \p name contains invalid characters.
+     * @return true when header is added successfully.
```

Comment:
Use either `@retval` with two values, or a single `@return`. We prefer `@return` for boolean values.
Make sure that you backtick `true` and `false`

Reply:
So, I would write it as ```@return `true` when header is added successfully, `false` otherwise.```